SbtPlugin/src/main/scala/com/dslplatform/sbt/Actions.scala

```diff
@@ -208,9 +209,13 @@ object Actions {
     if (namespace.nonEmpty) {
       ctx.put(Namespace.INSTANCE, namespace)
     }
+
+    val tempFolder = IO.createTemporaryDirectory
+    ctx.put("source:revenj.scala", tempFolder.getAbsolutePath)

```

Comment:
source:${target.toString} should be here instead?
Better yet... if we are calling this only for scala... should we pass in temp folder from the outside?
Are those temp folders cleaned up from somewhere?